src/fivetran-destination/src/destination/dml.rs

```diff
@@ -133,6 +138,11 @@ async fn write_batch(request: WriteBatchRequest) -> Result<(), anyhow::Error> {
 
     let (_dbname, client) = config::connect(request.configuration).await?;
 
+    // Note: This ordering of operations is important! Fivetran expects that we run "replace",
+    // "update", and then "delete" ops.
+    //
+    // See: https://materializeinc.slack.com/archives/C060KAR4802/p1706726115147059?thread_ts=1706720574.377339&cid=C060KAR4802
```

Comment:
Possibly this thread will be deleted. Could it get transcribed to github issue, gist, or described here with detail?

Reply:
Yeah good call, the message doesn't contain much other than just "you need to use this order", so I mentioned that here.